builder/internals.go

```diff
@@ -489,8 +489,11 @@ func (b *Builder) processImageFrom(img *imagepkg.Image) error {
 		b.Config = img.Config
 	}
 
-	if len(b.Config.Env) == 0 {
-		b.Config.Env = append(b.Config.Env, "PATH="+daemon.DefaultPathEnv)
+	// No default path on Windows (set by HCS)
+	if runtime.GOOS != "windows" {
+		if len(b.Config.Env) == 0 {
+			b.Config.Env = append(b.Config.Env, "PATH="+daemon.DefaultPathEnv)
```

Comment:
That's a fair question. The default path will be automatically set by the OS instance in the container - the best docker could do is (probably incorrectly) guess it wrongly. ENV in Dockerfile (or command line) can be used to override it where the user knows the container context.